FlowTests/FutureUtilitiesTests.swift

```diff
@@ -97,6 +97,20 @@ class FutureUtilitiesTests: FutureTest {
         }
     }
 
+    func testReplaceWithResultOnCurrentScheduler() {
+        let e = expectation(description: "value")
```

Comment:
nit: I see that this is the expectation description in other tests but I think it can be a bit more "descriptive" :P 
```suggestion
        let e = expectation(description: "initial future is disposed")
```